airflow/providers/apache/hive/hooks/hive.py

```diff
@@ -661,6 +661,19 @@ def get_partitions(
                 pnames = [p.name for p in table.partitionKeys]
                 return [dict(zip(pnames, p.values)) for p in parts]
 
+    def get_partition_by_name(self, db_name: str, tbl_name: str, part_name: str) -> Any:
+        """Get a metastore partition object
+
+        >>> hh = HiveMetastoreHook()
+        >>> tp = hh.get_partition_by_name(db_name='org_db', tbl_name='my_year_partitioned_table', part_name='year=2020')
```

Comment:
This line is too long which is why the test is failing
`airflow/providers/apache/hive/hooks/hive.py:668:0: C0301: Line too long (120/110) (line-too-long)`